apps/src/studio/Item.js

```diff
@@ -2,6 +2,7 @@ import Collidable from './collidable';
 import * as constants from './constants';
 import _ from 'lodash';
 import { ShakeActor } from './spriteActions';
+import Studio from './studio';
```

Comment:
Yes, but it seems to get set somewhere that doesn't run in these unit tests. This file is completely untested :crying_cat_face: